index.css

```diff
@@ -1,466 +1,709 @@
+@import url("https://fonts.googleapis.com/css2?family=Poppins:wght@500&display=swap");
```

Comment:
Currently, Poppins font is not showing up on your project, that's why it looks different than the Figma template. Font-weight:500 is the only option if you notice the URL you are importing. You can go to Google Fonts and add the correct font-weights. 👍